app/controllers/phrasing_phrases_controller.rb

```diff
@@ -6,7 +6,7 @@ class PhrasingPhrasesController < Phrasing.parent_controller.constantize
 
   include PhrasingHelper
 
-  before_filter :authorize_editor
+  before_action :authorize_editor
```

Comment:
If that's so, I'm okay with removing the Rails 3.2.0 support from travis. If this is just a style change, I would leave it as a before filter.